pyproject.toml

```diff
@@ -23,6 +23,12 @@ pyhelper-utils = "^0.0.12"
 [tool.poetry.group.dev.dependencies]
 ipdb = "^0.13.13"
 
+
```

Comment:
[tool.coverage.run]
omit = [path to tests]

[tool.coverage.report]
fail_under = 60
skip_empty = true

[tool.coverage.html]
directory = ".tests_coverage"